actionview/rollup.config.test.js

```diff
@@ -0,0 +1,16 @@
+import commonjs from "@rollup/plugin-commonjs"
```

Comment:
Maybe add a documentation block explaining why this file exists and how it fits into the testing process.

Reply:
I have added a comment to help explain this.